blog/2023-09-21-context-schema-and-export-to-external-editors/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Context schema and export to CodeSandbox or StackBlitz
+description: You can now open your machine in CodeSandbox or StackBlitz. These options show when you select either JavaScript or TypeScript export formats in the Code panel.
+authors: [laura]
+tags: [changelog, new, improved, fixed]
+date: 2023-09-21
+slug: 2023-09-21-context-schema-and-export-to-external-editors
+image: /blog/2023-09-21-context-schema-and-export-to-external-editors.png
+---
+
+You can now open your machine in CodeSandbox or StackBlitz. These options show when you select either JavaScript or TypeScript export formats in the **Code** panel. When you open your machine in these external editors, it will be deployed as a demo app so you can jump quickly into prototyping and get an example of how to integrate your machine into your codebase.
```

Comment:
Nit: The order of the features doesn't match the order in the title. Should the opening paragraph about exports get its own header and then be moved under the Context schema section?

Reply:
The order in the heading is just for readability. Export formats are best to be first as I think they’ll be more popular with more users for now.

Reply:
Understood. Thanks!